MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -502,6 +504,10 @@ - (void)checkVideoData {
             }
         }
     }
+    if ([self IsTesting]) {
```

Comment:
I'm not sure about adding this type of testing only code into our SDK - can you give me some context here?

Reply:
This will only execute when the tests are running, the Test scheme has an argument with the flag "TEST" that will be recognized here: 
`-(BOOL)IsTesting {
    return [[NSProcessInfo processInfo].arguments containsObject:@"TEST"];
}`
This is because with the change of logic the renditionChange tests were failing because we are checking if playback has started, with this flag we can change the state of "_started" to true to fulfill the check I mentioned before but the change of state is only happening when we are running the unit tests because the argument was just added in that scheme.
This same approach we have it in Kaltura, we use the same argument on the tests to assign the mockedDispatcher or the normal dispatcher.